Marlin/src/lcd/extui/lib/mks_ui/pic_manager.cpp

```diff
@@ -55,9 +55,10 @@ static const char assets[][LONG_FILENAME_LENGTH] = {
   "bmp_preHeat.bin",
   "bmp_extruct.bin",
   "bmp_mov.bin",
-  // "bmp_Zero.bin",
+  // "bmp_zero.bin",
   "bmp_leveling.bin",
   "bmp_filamentchange.bin",
+  "bmp_more.bin",
```

Comment:
Are these new binaries all present somewhere for users to deploy?

Reply:
Yes, it's automatic. This is the asset load list. There's a script `buildroot/share/PlatformIO/scripts/download_mks_assets.py` , that will download and prepare the assets in a simple folder for the user just copy to the SD.